app/controllers/api_controller.rb

```diff
@@ -49,4 +50,47 @@ def page_preview
     }
   end
 
+  # Send content for a page
+  def pages_content
+    page = Page.find_by(id: params[:id])
+    unless page
+      return render json: { errors: [ { detail: "Unable to find page" } ] }, status: 404
+    end
+
+    # Get content
+    content = params[:content]
+    unless content
+      return render json: { errors: [ { detail: "content paramter not provided" } ] }, status: 400
+    end
+
+    # Check page content
+    page.custom_html(content)
+    changes = PollPage.perform(page: page)
+
+    puts changes.inspect
+
+    if changes
+      render json: changes
+      Change.check
```

Comment:
`Change.check` will trigger a full check of all pages. It would be great to just check this one. I don't think there is currently a method to do that, but perhaps sticking that in `PageSnapshot#ensure_change` would be appropriate.